package.json

```diff
@@ -92,11 +92,13 @@
     "@types/pikaday": "1.7.4",
     "core-js": "^3.0.0",
     "dompurify": "^2.1.1",
-    "hot-formula-parser": "^4.0.0",
     "moment": "2.24.0",
     "numbro": "2.1.2",
     "pikaday": "1.8.0"
   },
+  "optionalDependencies": {
```

Comment:
@wojciechczerniak What we do with that? An optional dependency is always installed unless you run `npm install --no-optional`.

Reply:
I'm ok with either. IMO not a big deal that it is installed by default, it won't be included in the application bundle unless imported by the developer. If devs don't wan't optional dependencies and thus avoid downloading unnecessary code they will use `--no-optional` flag anyway. Otherwise, the NPM is installing a lot of optional stuff all the time.